tpcx_bb/benchmark_runner/draco/run.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+PROJECT=sw_rapids_testing
+PARTITION=batch_32GB_short
+IMAGE=randerzander/draco:8-24_0
```

Comment:
Do we expect this to be the stable name? This is date-tagged

Reply:
No, this is not intended to be be "stable", just useful for development. I'll add a note to the README about it.

If we ever do start publishing a tpcx-bb container, this would be a good place to put it to use though.